### PR TITLE
add TLM connection opening from Login.jsx

### PR DESCRIPTION
**Pull request by Skylsmoi**
https://github.com/tracim/tracim/issues/2850

## Checkpoints

The goal of this section is to help code reviewers and the merge request author to do the required checks on this pull request. Please don't edit this list.
If one or more of these checkpoints are out of scope, please write below why they are.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, then there are associated issues and issue is referenced in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present
- [x] Automated tests have been written (covering the feature or non regression), or *at least* an issue has been created to implement the test

**For developers**

- [x] Manual tests done to ensure stability on whole application layers, issue expectation follow
- [x] The original issue has been updated with a short summary of the implemented solution

**For the quality team**

- [x] Tested by the quality team

### CODE REVIEW

**Comment by Skylsmoi** [comment]
What to test here :
- Test that the connection to the TLM process works on both login ways:
  - first load Tracim while being connected (you end up on the home page)
  - first load Tracim without being connected (you end up on the login page)

File: frontend/src/container/Header.jsx [hunk 1]
```diff
@@ -16,7 +16,8 @@ import logoHeader from '../img/logo-tracim.png'
 import {
   newFlashMessage,
   setUserLang,
-  setUserDisconnected
+  setUserDisconnected,
+  resetLiveMessageManager
```
Comment (gbonaspetti) [inline frontend/src/container/Header.jsx:20]: Unused import?

Comment (Skylsmoi) [reply frontend/src/container/Header.jsx:20]: indeed

File: frontend/src/reducer/tlmManager.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { SET, TLM_MANAGER } from '../action-creator.sync.js'
+
+export default function tlmManager (state = null, action) {
+  switch(action.type) {
```
Comment (metzgegu) [inline frontend/src/reducer/tlmManager.js:4]: Need a space after `switch`

**Comment by metzgegu** [comment]
Verified on branch.

**Pull request merged**